addons/binding/org.openhab.binding.irtrans/src/main/java/org/openhab/binding/irtrans/handler/TransceiverStatusListener.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.irtrans.handler;
+
+import org.openhab.binding.irtrans.IRcommand;
+
+/**
+ * The {@link TransceiverStatusListener} is interface that is to be implemented
+ * by all classes that wish to be informed of events happening to a infrared
+ * transceiver
+ *
+ * @author Karel Goderis - Initial contribution
+ * @since 2.1.0
+ *
+ */
+public interface TransceiverStatusListener {
+
+    /**
+     *
+     * Called when the ethernet transceiver/bridge receives an infrared command
+     *
+     * @param bridge
+     * @param command - the infrared command
+     */
+    public void onCommandReceived(EthernetBridgeHandler bridge, IRcommand command);
+
+    /**
+     *
+     * Called when the connection with the remote transceiver is lost
+     *
+     * @param bridge
+     */
+    public void onBridgeDisconnected(EthernetBridgeHandler bridge);
+
+    /**
+     * Called when the connection with the remote transceiver is established
+     *
+     * @param bridge
+     */
+    public void onBridgeConnected(EthernetBridgeHandler bridge);
+
+}
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

It is also missing from the parent pom.xml